README.md

```diff
@@ -18,47 +18,51 @@ This boilerplate provides a skeleton to start developing cross-browser web exten
 ## Development
 
 ```bash
-npm i
+npm install -g pnpm
```

Comment:
I think we don't need this. We can say like "You need to install pnpm and put the link of the doc of pnpm here". What do you think?

Reply:
yes, thanks